Composer/packages/client/src/App.tsx

```diff
@@ -38,6 +45,19 @@ export const App: React.FC = () => {
     ipcRenderer?.on('cleanup', (_event) => {
       performAppCleanupOnQuit();
     });
+
+    let days = surveyStorage.get('days', 0);
```

Comment:
All of this logic should be handled in the `setSurveyEligibility` dispatcher. You should just call the dispatcher on mount and let it determine how and if it should update update the value.

Reply:
Good idea. I'll fix that.